### DISCUSSION TITLE
How much of the data generated by IoT devices is actually used, and how much is wasted? Why is so much data ignored?

### DISCUSSION THREAD

**Discussion by wilsonmar** [microsoft/IoT-For-Beginners#224] [Lessons]
From https://github.com/microsoft/IoT-For-Beginners/blob/main/1-getting-started/lessons/1-introduction-to-iot/README.md

https://www.cisco.com/c/en/us/solutions/executive-perspectives/annual-internet-report/index.html
(viewed 17 Jul 2021) predicted that by the end of 2019, "IoT will generate more than 500 zettabytes per year in data—and in the years beyond, that number is expected to grow exponentially, not linearly."

One zettabyte (1 ZB) is one thousand exabytes or one billion terabytes (TB).

**Comment by wilsonmar** [comment]
https://www.gooddata.com/blog/80-percent-of-companies-will-fail-to-monetize-iot-data-according-gartner/

**Reply by jimbobbennett** [reply]
It's amazing just how much data will be ignored and how many opportunities to use this that will be missed.

Or conversely, how much is spent buying hardware and gathering data that will be completely ignored.

Which leads to an interesting question - is it better to gather too much ad ignore, or not gather it and hope you don't need it someday.
